src/mprk.jl

```diff
@@ -510,6 +510,8 @@ function perform_step!(integrator, cache::MPRK22ConstantCache, repeat_step = fal
 
     # Attention: Implementation assumes that the pds is conservative,
     # i.e. , P[i, i] == 0 for all i
+    # We use P2 to store the last evaluation of the PDS 
+    # as well as to store the system matrix of the linear system
```

Comment:
It's still `M` below in the out-of-place version here. Should this comment be moved to the in-place version?